02_calculator/calculator.rb

```diff
@@ -0,0 +1,31 @@
+def add(a,b)
+  a+b
+end
+
+def subtract(a,b)
+  return (a-b)
+end
+
+def sum (arr)
+  sum=0.0
+  if arr.length<=0 then
+    return  0
+  end
```

Comment:
Why do you need this `if` here. Does it matter if you remove it or keep it for the output of the function ? (aka do tests continue to pass?)

-- try to minimize exceptional cases when you write code